README.md

```diff
@@ -156,6 +156,22 @@ cd PVM && git pull
 sudo reboot
 ```
 
+## Logs
+
+All logs for each time each RPI is stored in the log folder.
```

Comment:
I know we've been using that name among ourselves but lets's make sure that the first time `RPI` appears in the readme it will be connected to it's meaning. like: `to install X on your Raspberry Pi (RPI)`. from then on the reader shall know that we can refer to it as either `RPI` or `Raspberry Pi` and we'll be good 😉